contribs/gmf/apps/desktop/index.html

```diff
@@ -333,6 +349,17 @@
         module.constant('gmfTreeUrl', 'https://geomapfish-demo.camptocamp.net/2.2/wsgi/themes?version=2&background=background&interface=desktop');
         module.constant('gmfLayersUrl', 'https://geomapfish-demo.camptocamp.net/2.2/wsgi/layers/');
         module.constant('gmfShortenerCreateUrl', 'https://geomapfish-demo.camptocamp.net/2.2/wsgi/short/create');
+        module.constant('gmfOsrmBackendUrl', 'http://routing.osm.ch/');
+        module.constant('gmfRoutingProfiles', [
+          {label : 'Car', profile: 'routed-car'},
+          {label : 'Bike (City)', profile: 'routed-bike'},
+          {label : 'Bike (Touring)', profile: 'routed-bike-touring'},
+          {label : 'Foot', profile: 'routed-foot'},
+          {label : 'Hiking', profile: 'routed-hiking'}
+        ]);
+        module.constant('gmfRoutingSearchDefaultParams', {
+          'countrycodes': 'CH'
+        });
```

Comment:
Instead of having multiple options why not have a single object `gmfRoutingOptions` with everything inside?